client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getPostRevision,
+	normalizeForDisplay,
+} from 'state/posts/revisions/selectors';
+
+class EditorDiffViewer extends PureComponent {
```

Comment:
For now a functional component would work, I tend to start with components while building as it allows to quickly add methods and lifecycle-related stuff without updating a lot of lines / re-indenting stuff around (now I wonder if there isn't an atom or emacs plugin that would allow me to switch between a component form and a functional one in a few keystrokes).